node-feature-discovery-job.json.template

```diff
@@ -17,6 +17,14 @@
         }
       },
       "spec": {
+        "volumes": [
```

Comment:
also the mnt namespace gives the container its own view of the “mounts” on the system resulting in a different views of the filesystem hierarchy